notebooks/llm-agent-functioncall/llm-agent-functioncall-qwen.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    "qwen-agent&gt;=0.0.6" "transformers&gt;=4.38.1" "gradio==4.21.0", "modelscope-studio&gt;=0.4.0" "langchain&gt;=0.2.3" "langchain-community&gt;=0.2.4" "wikipedia"
</pre><p>is there any specific reasons why you use == for gradio version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Because as my test, only this version is compatible with ModelScope-stuido, and other version may have different kind of BUG.

https://github.com/QwenLM/Qwen-Agent/blob/main/setup.py#L70